internal/pkg/test/envtest.go

```diff
@@ -170,8 +170,8 @@ func PrepareEnvTest(controllers []manager.Registerer, namespaces []string, baseP
 func TeardownEnvTest(testEnv *envtest.Environment, cancel context.CancelFunc) {
 	cancel()
 	By("tearing down the test environment")
-	err := testEnv.Stop()
-	Expect(err).NotTo(HaveOccurred())
+	_ = testEnv.Stop()
+	// Expect(err).NotTo(HaveOccurred())
```

Comment:
Could you please explain why skipping this ?

Reply:
ah .. it's related to a flaky test, see also my failed fix attempt here https://github.com/netobserv/network-observability-operator/pull/1950  
basically that line sometimes fail since a couple of months, with the kube env never stopping (even if I increase the timeout), I haven't found the root cause so far and was quite annoyed ... but I'll remove the change, and will try harder .. later :-)

Reply:
reverted